src/github/mod.rs

```diff
@@ -150,6 +151,43 @@ const IGNORED_DIRECTORIES: &[&str] = &[
     "debug",
 ];
 
+const ALLOWED_LICENCES: &[&str] = &[
+    "0bsd",
+    "apache-2.0",
+    "bsd-2-clause",
+    "bsd-3-clause",
+    "bsd-3-clause-clear",
+    "bsd-4-clause",
+    "isc",
+    "mit",
+    "mpl-2.0",
```

Comment:
`mpl-2.0` is [a weak-copy left license](https://www.mozilla.org/en-US/MPL/2.0/FAQ/) which (in theory) comes with clauses restricting our ability to distribute the code or use it _within_ our code. That all is abit hairy with AI interpretation of source code and since we're being overly cautious at first, let's just leave this one out.

Reply:
https://github.com/open-sauced/repo-query/pull/9/commits/9ef6aacbb1083ad9602a11b74ce03103b827569c